src/main/java/com/epam/flightscontroltool/control/mapper/AirplaneMapper.java

```diff
@@ -1,14 +1,14 @@
-package com.epam.flightscontroltool.domain.airplane.mapper;
+package com.epam.flightscontroltool.control.mapper;
 
-import com.epam.flightscontroltool.controller.dto.airplane.AirplaneDto;
-import com.epam.flightscontroltool.domain.airplane.entity.Airplane;
-import org.mapstruct.InheritInverseConfiguration;
+import com.epam.flightscontroltool.boundary.dto.AirplaneDto;
+import com.epam.flightscontroltool.boundary.request.AirplaneRequest;
+import com.epam.flightscontroltool.entity.Airplane;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
 import org.mapstruct.ReportingPolicy;
 import org.mapstruct.factory.Mappers;
 
-@Mapper(componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.IGNORE)
+@Mapper(unmappedTargetPolicy = ReportingPolicy.IGNORE)
 public abstract class AirplaneMapper {
```

Comment:
Not 100% sure. But replace abstract class and with interface instead. It should work. Doing this by using abstract classes is an old way.